util/aligned_buffer.h

```diff
@@ -70,7 +78,7 @@ class AlignedBuffer {
     return cursize_;
   }
 
-  const char* BufferStart() const {
+  char* BufferStart() const {
```

Comment:
const overload must return const ptr. Make a separate non-const overload to return non-const ptr